pycircos/pycircos.py

```diff
@@ -5,7 +5,6 @@
 import math
 import urllib
 import tempfile
-import requests
```

Comment:
`requests` library is not used in the code and there is no installation description in the setup.py. I removed it because it is also a cause of errors during automatic API documentation generation.